packages/react-components/react-toolbar/src/stories/Toolbar/ToolbarWithPopover.stories.tsx

```diff
@@ -4,40 +4,106 @@ import type { ToolbarProps } from '@fluentui/react-toolbar';
 import { Popover, PopoverSurface, PopoverTrigger, Button } from '@fluentui/react-components';
 import { CalendarMonthRegular, Accessibility24Filled } from '@fluentui/react-icons';
 
-export const WithPopover = (props: Partial<ToolbarProps>) => (
-  <Toolbar {...props} size="small">
-    <Popover withArrow>
-      <PopoverTrigger disableButtonEnhancement>
-        <ToolbarButton appearance="primary">See more...</ToolbarButton>
-      </PopoverTrigger>
-      <PopoverSurface>
-        <div>
-          <h3>Popover content</h3>
-          <Button>Action</Button>
-        </div>
-      </PopoverSurface>
-    </Popover>
-    <ToolbarDivider />
-    <Popover withArrow>
-      <PopoverTrigger disableButtonEnhancement>
-        <ToolbarButton aria-label="Popover trigger - Calendar" icon={<CalendarMonthRegular />} />
-      </PopoverTrigger>
-      <PopoverSurface>
-        <div>
-          <h3>Popover content</h3>
-          <Button>Action</Button>
-        </div>
-      </PopoverSurface>
-    </Popover>
-    <Popover withArrow>
-      <PopoverTrigger disableButtonEnhancement>
-        <ToolbarButton aria-label="Popover trigger - Accessibility" icon={<Accessibility24Filled />} />
-      </PopoverTrigger>
-      <PopoverSurface>
-        <div>
-          <h3>Popover content - Accessibility</h3> <Button>Action</Button>
-        </div>
-      </PopoverSurface>
-    </Popover>
-  </Toolbar>
-);
+export const WithPopover = (props: Partial<ToolbarProps>) => {
+  const [open, setOpen] = React.useState({
```

Comment:
it makes no sense to have multiple popovers open at the same time. the state here should be an enum `first` `second` or `third`